lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -165,6 +165,7 @@ func (s *builderServicer) Build(ctx context.Context, request *builder_protos.Bui
 			ServiceAreaMaps:          getServiceAreaMaps(nwEpc.ServiceAreaMaps),
 			FederatedModeMap:         getFederatedModeMap(federatedNetworkConfigs),
 			CongestionControlEnabled: swag.BoolValue(gwEpc.CongestionControlEnabled),
+			SentryConfig:             &lte_mconfig.SentryConfig{},
```

Comment:
I'm guessing you'll have a separate PR for actually filling these out right?

Reply:
Yes I'll have it in the next PR.

Reply:
Okay so, big question for @themarwhal and @waqaraqeel: will the SentryConfig ever be different *per-service*? Because if *no*, then we shouldn't be sending down copies of the same config to each service that needs it (since, you know, each service will need it)

And we *definitely* shouldn't be sending it down to just 1 arbitrary+non-core service then having that value apply across multiple services, especially cross-language -- so like, don't send it to `mme` then apply it to `magmad` and `bootstrapper`, that violates the module dependency tree https://magma.github.io/magma/docs/next/orc8r/dev_dependencies

So if it is the case that we really just want 1 normalized SentryConfig sent down, perhaps it's time to support per-module pseudo services. In this case, either use an `orc8r` pseudo service to hold the SentryConfig, or, if you want to maintain backwards compat, use the `magmad` service

But I think it's a messy, messy idea to mix and match like (it seems) we're doing now. Lmk if I'm misunderstanding though 🙂

Reply:
Hey @hcgatewood! To your first point, the plan for now was to only stream the config to MME and SessionD services only for now. They are the services that have been benefitting most from Sentry, so I don't want to enable it for the Python services just yet. (Which brings me to think that we should probably remove `url_python` from the swagger API to not expose it until we start using it.)

To your second point, we are not aiming to send it to one service and apply it to multiple.

My thought is, if we keep going with this approach of sending the config down per-service, we should split up the mconfig object so that we only send what we need for each service.
`SessionD` -> `url_native` and `sample_rate`
`MME` -> `url_native`, `sample_rate`, and `upload_mme_log`

I also thought about sending the sentry config under a global?non-service specific key, but since we don't use the config everywhere yet, it may not be worth the trouble just yet.

Reply:
I agree @themarwhal. It would make sense to move to `magmad` or another cross-service config solution when we enable Sentry for all/more services. We could eliminate the unified `SentryConfig` proto structure and send individual fields to services, but I think the unified structure is actually be cleaner unless we care about every byte of config we send.